src/app/auth/auth.token.interceptor.service.ts

```diff
@@ -13,17 +13,22 @@ import { tap } from 'rxjs/operators';
 import { Router } from '@angular/router';
 
 import { NbAuthJWTToken, NbAuthService } from '@nebular/auth';
-
-import { environment } from 'environments/environment';
+import { environment } from '../../environments/environment';
 
 @Injectable()
 export class TokenInterceptor implements HttpInterceptor {
 
+  loginUrl = '/auth/login';
```

Comment:
`/auth/login` is used in different places. You can probably define it in envoronments/environment.ts.